lib/common_test/src/ct_ssh.erl

```diff
@@ -283,71 +299,80 @@ session_close(SSH, ChannelId) ->
 exec(SSH, Command) ->
```

Comment:
Just above: ` %%% @equiv exec(SSH,Command,DefaultTimeout)` and I spec'd `exec/3`.
So: intentionally not spec'd.